src/index.ts

```diff
@@ -0,0 +1,2 @@
+export { fromObservable as default, fromObservable } from "./fromObservable";
```

Comment:
fromObservable gets exported both as a named export and as a default export in order not to break compatibility, but the default export should be deprecated in order to maintain symmetry.